sdk/storage/azure-storage-blob/azure/storage/blob/_models.py

```diff
@@ -836,11 +836,81 @@ class PageRange(DictMixin):
         Start of page range in bytes.
     :param int end:
         End of page range in bytes.
+    :ivar bool cleared:
+        Whether the range has been cleared.
     """
 
-    def __init__(self, start=None, end=None):
+    def __init__(self, start=None, end=None, *, cleared=False):
```

Comment:
Was this object previously used for any APIs apart from `get_page_ranges`?
And if I use this with the previous deprecated API, will I get `cleared=False` on my output when that may not be true?

Reply:
I couldn't actually find anywhere this existing class was used besides being exported. It's not used in the output of the existing `get_page_ranges()` as that just returns a raw Tuple. I also don't see it in any tests, samples, or as the input to anything. Searching the code, its only defined and exported.

So I decided it was safe to just modify and use it.

Reply:
Sounds like lemonade to me.